tendrl/node_agent/manager/pull_hardware_inventory.py

```diff
@@ -122,6 +122,98 @@ def getTendrlContext():
     return tendrl_context
 
 
+def getNodeDisk():
+    '''returns structure
+
+    {"nodename": [{"DevName":   "devicename",
+                  "FSType":     "fstype",
+                  "FSUUID":     "uuid",
+                  "Model":      "model",
+                  "MountPoint": ["mountpoint", ...],
+                  "Name":       "name",
+                  "Parent":     "parentdevicename",
+                  "Size":       uint64,
+                  "Type":       "type",
+                  "Used":       boolean,
+                  "SSD":        boolean,
+                  "Vendor":     "string",
+                  "DiskId":     "uuid"}, ...], ...}
+    '''
+    devlist = {}
+    dev_info = {}
+    rv = []
+    columes = 'NAME,KNAME,FSTYPE,MOUNTPOINT,UUID,PARTUUID,MODEL,SIZE,TYPE,' \
+              'PKNAME,VENDOR'
+    keys = columes.split(',')
+    lsblk = ("lsblk --all --bytes --noheadings --output='%s' --path --raw" %
```

Comment:
@GowthamShanmugam Even if the package exists by default, still add a dependency on it because you actually have a direct dependency on it. Always ensure that all direct dependencies are explicitly specified.